--- tl2.c.orig	2008-09-09 00:41:02.000000000 +0200
+++ tl2.c	2014-06-23 19:12:44.820990756 +0200
@@ -116,7 +116,11 @@
 #  define PROF_STM_WRITELOCAL_END()     /* nothing */
 #  define PROF_STM_SUCCESS()            /* nothing */
 
+#include "eventlog.h"
 
+#if LWM_TYPE == LWM_EVT_LOG_PER_THREAD_ABORTS && LWM_TRACK_ABORTS == 1
+  #define PROF_STM_ABORT_BEGIN() LOG_EVENT(Self->UniqID, TX_ABORT, __COUNTER__)
+#endif
 
 typedef int            BitMap;
 typedef uintptr_t      vwLock;  /* (Version,LOCKBIT) */
